R/singlecell_alignIndex.R

```diff
@@ -16,7 +16,7 @@
 #'  )
 #' }
 #' @export
-singlecell_alignIndex <- function(input_dir_path, genome_dir_path, bamsave=TRUE){
+singlecell_alignIndex <- function(input_dir_path, genome_dir_path, bamsave="true"){
```

Comment:
I'd prefer to keep it as a logical type, and then transforming it to a string when building the argument for the docker to expose a _nice_ interface to the users of the library